SD2019StackOverflowApp/src/main/java/ro/utcn/sd/mca/SD2019StackOverflowApp/persistence/jdbc/FindAnswerByIdJdbc.java

```diff
@@ -0,0 +1,35 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp.persistence.jdbc;
+
+import org.springframework.jdbc.core.RowMapper;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.entity.Answer;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+class FindAnswerByIdJdbc implements JdbcSpecification<Answer> {
+    private Integer answerId;
+
+    FindAnswerByIdJdbc(Integer answerId) {
+        this.answerId = answerId;
+    }
+
+    @Override
+    public String getSqlCommand() {
+        return "SELECT * FROM answer WHERE answer_id=?";
+    }
+
+    @Override
+    public List<Object> getSqlCommandArgsValues() {
+        List<Object> l = new ArrayList<>();
+        l.add(answerId);
+        return l;
+    }
+
+    @Override
+    public RowMapper getRowMapper() {
```

Comment:
The RowMappers are heavily duplicated. Basically, you could extract a `QuestionRowMapper` class, an `AnswerRowMapper` class, etc.